### PR TITLE
Model Book finished

### PR DESCRIPTION
**Pull request by blassoto**
### Summary

- Add model Book

### Trello Card

https://trello.com/c/nuVxWk4Y

### CODE REVIEW

File: app/models/book.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class Book < ApplicationRecord
+  validates :title, presence: true
+  validates :author, presence: true
+  validates :image, presence: true
+  validates :editor, presence: true
+  validates :genere, presence: true
```
Comment (joredudiaz) [inline app/models/book.rb]:
```suggestion
  validates :genre, presence: true
```

File: db/migrate/20200326160651_create_books.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class CreateBooks < ActiveRecord::Migration[5.1]
+  def change
+    create_table :books do |t|
+      t.string :title, null: false
+      t.string :author, null: false
+      t.string :image, null: false
+      t.string :editor, null: false
+      t.string :genere, null: false
```
Comment (joredudiaz) [inline db/migrate/20200326160651_create_books.rb]:
```suggestion
      t.string :genre, null: false
```

File: app/models/book.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+  validates :year, presence: true
```
Comment (aleochoam) [inline app/models/book.rb]: Normalmente ponermos las validaciones iguales en una sola línea

```suggestion
  validates :title, :author, :image, :editor, :genere, :year, presence: true
```

File: app/models/book.rb (hunk 1 above)
Comment (blassoto) [reply app/models/book.rb]: In the branch test_book I modified it

File: db/migrate/20200326160651_create_books.rb (hunk 2 above)
Comment (blassoto) [reply db/migrate/20200326160651_create_books.rb]: In the branch test_book I modified it

**Comment by blassoto** [comment]
@aleochoam 
imagen

File: app/models/book.rb (hunk 1 above)
Comment (blassoto) [reply app/models/book.rb]: Thanks, modified in a commit

File: db/migrate/20200326160651_create_books.rb (hunk 2 above)
Comment (blassoto) [reply db/migrate/20200326160651_create_books.rb]: Thanks, modified in a commit

**Comment by blassoto** [comment]
@aleochoam 
imagen

File: db/schema.rb [hunk 4]
```diff
@@ -15,6 +15,17 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "books", force: :cascade do |t|
+    t.string "title", null: false
+    t.string "author", null: false
+    t.string "image", null: false
+    t.string "editor", null: false
+    t.string "genere", null: false
```
Comment (joredudiaz) [inline db/schema.rb]: you change the field on the model and the migration but at the schema is the same, run migration again to change this

Comment (blassoto) [reply db/schema.rb]: done!

**Pull request merged**